src/route/routes.js

```diff
@@ -7,10 +7,10 @@ export const routes = {
         routerPath: '/search',
         getPathWithParams: (id = '') => `/search?id=${id}`
     },
-    // error404: {
-    //     routerPath: '*',
```

Comment:
should be:
```
routerPath: '*'
```

Reply:
ok

Reply:
@wikbro7650 you didn't changed that as @PoProstuMieciek expected, you changed it to '', instead of '*', also it's still not working on preview in PR

Reply:
Sorry, missclick. I'll change it now